OpenOversight/tests/test_functional.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+
+def test_user_can_load_homepage_and_get_to_form(mockdata, browser):
+    browser.get("http://localhost:5000")
+
+    # Complainant loads homepage
+    assert "OpenOversight" in browser.title
+    browser.find_element_by_link_text("Try it").click()
+    time.sleep(5)
```

Comment:
Word, good feedback here. To address this in commit 167804a I made helper function `wait_for_element` for the functional tests to wait for an element of interest to be present or throw a selenium `TimeoutException` and fail the test. I also added a context manager `wait_for_page_load()` for page clicks such that we wait for the new page to finish loading (I got that from [this page](http://www.obeythetestinggoat.com/how-to-get-selenium-to-wait-for-page-load-after-a-click.html)).